### PR TITLE
Release 0.6.1

### PR DESCRIPTION
**Pull request by roidelapluie**
Signed-off-by: Julien Pivotto <roidelapluie@inuits.eu>

### CODE REVIEW

**Comment by roidelapluie** [comment]
I actually think we should drop VerifyClientCertIfGiven RequireAnyClientCert and RequestClientCert.

**Comment by LeviHarrison** [comment]
Ehhhhhhh we refer to https://golang.org/pkg/crypto/tls/#ClientAuthType for the types so I think we should support all of them.

**Comment by LeviHarrison** [comment]
I guess we could move that documentation into our own repo and eliminate those options

**Comment by roidelapluie** [comment]
So, should we go ahead with this? We support all of them but users should be aware that only one is secure.

**Comment by LeviHarrison** [comment]
Sure

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## 0.6.1 / 2021-06-30
+
+* [BUGFIX] Allow RequireAnyClientCert as client_auth_type.
```
Comment (LeviHarrison) [inline CHANGELOG.md:3]: Should we add the PR number?

**Review by LeviHarrison — APPROVED** [review]
Looks good

**Pull request merged**